### DISCUSSION TITLE
What does the "iOS Peloton workaround" option do?

### DISCUSSION THREAD

**Discussion by lychee128** [cagnulein/qdomyos-zwift#312] [Q&A] [answered]
Thanks.

**Comment by cagnulein** [comment, marked as answer]
Peloton needs special Bluetooth attributes to works from ios. This setting gives to it. Also, on the lata beta, it's used also for Zwift in order to run native swift code instead of c++ code: this fixes all the crash issue on zwift on ios

**Reply by lychee128** [reply]
Thanks
